CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - [#2056](https://github.com/wasmerio/wasmer/pull/2056) Change back to depend on the `enumset` crate instead of `wasmer_enumset`
 
 ### Fixed
+- [#2058](https://github.com/wasmerio/wasmer/pull/2058) `wasm_version_t` wasn't bound correctly to C.
```

Comment:
This message in the changelog isn't meaningful without context, perhaps change it to something that users can understand as it affects them